### ISSUE TITLE
Tensorboard 2.1: how to cast tf.keras.metrics that require mandatory argument as hp.Metric

### ISSUE THREAD

**Issue by mthomp89** [tensorflow/tensorboard#4161]
Some tf.keras.metrics require at least one mandatory argument or offer optional arguments. Examples are:

[tf.keras.metrics.PrecisionAtRecall](https://www.tensorflow.org/versions/r2.1/api_docs/python/tf/keras/metrics/PrecisionAtRecall)
[tf.keras.metrics.SpecificityAtSensitivity](https://www.tensorflow.org/versions/r2.1/api_docs/python/tf/keras/metrics/SpecificityAtSensitivity)
[tf.keras.metrics.AUC](https://www.tensorflow.org/versions/r2.1/api_docs/python/tf/keras/metrics/AUC)

I'd like to use these in order to evaluate HParam tuning. However, each of these requires additional arguments that are not allowed in defining the hp.Metric. For example, I am attempting to use the following:

```
tf.keras.metrics.PrecisionAtRecall(.75)
tf.keras.metrics.SpecificityAtSensitivity(.5)
tf.keras.metrics.AUC(curve='PR')
```

How and where do these additional arguments build into the tensorboard hp.Metrics?

tensorboard 2.1
python 3.6

**Comment by wchargin** [comment]
Hi @mthomp89! The TensorBoard hparams plugin can read any metrics that
are available on the TensorBoard scalars dashboard. My understanding is
that if your model uses the `tf.keras.callbacks.TensorBoard` callback,
then the metrics defined should show up as scalar charts. Is that right?
If so, you can register `hp.Metric`s with the same names as the tags on
those charts.

If you need further clarification, could you please post a short,
self-contained example of your Keras model that we can run to diagnose
the issue? Thanks!

**Comment by mthomp89** [comment]
I’ll be able to respond within 24 hours.

**Comment by rmothukuru** [comment]
Automatically closing due to lack of recent activity. Please update the issue when new information becomes available, and we will reopen the issue. Thanks!
